nixops_encrypted_links/lib.py

```diff
@@ -163,13 +157,6 @@ def do_machine(m: nixops.backends.MachineState) -> None:
             trusted_interfaces[m.name].add("tun" + str(local_tunnel))
             trusted_interfaces[m2.name].add("tun" + str(remote_tunnel))
 
-        private_ipv4 = m.private_ipv4
-        if private_ipv4:
-            attrs_list.append({("networking", "privateIPv4"): private_ipv4})
-        public_ipv4 = m.public_ipv4
-        if public_ipv4:
-            attrs_list.append({("networking", "publicIPv4"): public_ipv4})
-
```

Comment:
This is already contained in the upstream deployment file and if left here results in merge conflicts since nix_merge doesn't handle str, str merges: https://github.com/NixOS/nixops/blob/master/nixops/deployment.py#L669-L675